src/App.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import './App.scss';
+import { Person } from './components/Person/Person';
```

Comment:
if you create a separate folder `Person` with `Person.tsx`, better to create also `index.js` file with re-export:
```
export * from './Person.tsx;
```
then you'll be able to import from `index.js` file automatically, and import will look like
`import { Person } from './components/Person`;